local/kind/setup.sh

```diff
@@ -79,4 +86,9 @@ for cluster in "${CLUSTERS[@]}"; do
 
 done
 
+NO_ARGOCD="${NO_ARGOCD:-}"
+if [[ "${NO_ARGOCD,,}" != "true" ]]; then
+	KUBECONFIG="${TMP_DIR}/${CLUSTERS[0]}.kubeconfig" ../argocd/setup.sh
+fi
+
```

Comment:
Running `deploy.sh` requires the presence of `plnsvc.yaml` which is the kubeconfig on which to install the pipeline service.
Would you mind adding `ln -sf "${TMP_DIR}/${CLUSTERS[0]}.kubeconfig" "${TMP_DIR}/plnsvc.yaml"`?

Reply:
With the phase 1 approach the Pipelines service operators need to be installed on the workload clusters. There may be more than 1. I am looking at this as well. The integration with next steps can come in a separate PR.